### PR TITLE
feat(ObjectPage): Introduce Breadcrumbs and key infos

### PR DESCRIPTION
**Pull request by clemai**
- Add Breadcrumbs above object title, if given.
- Add key Info area.
- Hide header content area, if renderHeaderContent is not given.
- Fix lost title/subtitle issue, when switching from collapsed header to expanded header via expand button.

### CODE REVIEW

File: packages/main/src/components/ObjectPage/ObjectPage.test.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { ObjectPageMode } from '@ui5/webcomponents-react/lib/ObjectPageMode';
 import { ObjectPageSection } from '@ui5/webcomponents-react/lib/ObjectPageSection';
 import { ObjectPageSubSection } from '@ui5/webcomponents-react/lib/ObjectPageSubSection';
 import { Text } from '@ui5/webcomponents-react/lib/Text';
+import { Title } from '../../webComponents/Title';
```
[the remark this answers is not in the document]
Comment (clemai) [reply packages/main/src/components/ObjectPage/ObjectPage.test.tsx]: Done.

File: packages/main/src/components/ObjectPage/index.tsx [hunk 2]
```diff
@@ -72,7 +74,7 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
     image,
     subTitle,
     headerActions,
-    renderHeaderContent,
+    renderHeaderContent: renderHeaderContentParam,
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: renderHeaderContentProp?

File: packages/main/src/components/ObjectPage/index.tsx [hunk 3]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
     }
 
     if (showTitleInHeaderContent) {
-      const headerContents = renderHeaderContent();
+      const headerContents = renderHeaderContentParam && renderHeaderContentParam();
       let firstElement;
       let contents = [];
 
-      if (headerContents.type === React.Fragment) {
+      if (headerContents && headerContents.type === React.Fragment) {
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: `headerContents?.type === React.Fragment`

File: packages/main/src/components/ObjectPage/index.tsx [hunk 4]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
       } else {
         firstElement = headerContents;
       }
-
       return (
         <div className={classes.contentHeader}>
           <div className={classes.headerContent}>
-            <div>{avatar}</div>
-            <div>
+            <div style={{ display: 'flex', flexDirection: 'row' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 5]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+              <div>{avatar}</div>
               <div style={{ display: 'flex', flexDirection: 'column' }}>
-                <h1 className={classes.title}>{title}</h1>
-                <span className={classes.subTitle}>{subTitle}</span>
-                {firstElement}
+                <div>{renderBreadcrumbs && renderBreadcrumbs()}</div>
+                <div style={{ display: 'flex', flexDirection: 'row' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 6]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+                  <div style={{ display: 'flex', flexDirection: 'column' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 7]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+                    <h1 className={classes.title}>{title}</h1>
+                    <span className={classes.subTitle}>{subTitle}</span>
+                    <span> {firstElement}</span>
+                  </div>
+                  <div style={{ display: 'flex', flexDirection: 'row' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 8]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+                    {contents.map((c, index) => (
+                      <div key={`customContent-${index}`} style={{ marginLeft: '1rem' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 9]
```diff
@@ -258,30 +263,41 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+                        {c}
+                      </div>
+                    ))}
+                  </div>
+                  <div className={classes.keyInfos} style={{ display: 'flex', flexDirection: 'row' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 10]
```diff
@@ -303,25 +322,31 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
             <div style={{ display: 'flex', alignItems: 'center' }}>
               {image && collapsedHeader && !expandHeaderActive && !alwaysShowContentHeader && (
                 <div style={{ marginRight: '1rem' }}>
                   <CollapsedAvatar image={image} imageShapeCircle={imageShapeCircle} />
                 </div>
               )}
               <span className={classes.container}>
-                <h1 className={classes.title}>{title}</h1>
-                <span className={classes.subTitle}>{subTitle}</span>
+                <div style={{ display: 'flex', flexDirection: 'column' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 11]
```diff
@@ -303,25 +322,31 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+                  {renderBreadcrumbs && renderBreadcrumbs()}
+                  <div style={{ display: 'flex', flexDirection: 'row' }}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

File: packages/main/src/components/ObjectPage/index.tsx [hunk 12]
```diff
@@ -303,25 +322,31 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
...
+                    <h1 className={classes.title}>{title}</h1>
+                    <span className={classes.subTitle}>{subTitle}</span>
+                    <div style={{ display: 'flex', flexDirection: 'row' }} className={classes.keyInfos}>
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: inline object

**Pull request merged**